trident-use/trident-fsx.adoc

```diff
@@ -104,6 +104,8 @@ NOTE: Do not specify `dataLIF` for the `ontap-san` and `ontap-san-economy` drive
 
 WARNING: When using Amazon FSx for NetApp ONTAP with Astra Trident, the `limitAggregateUsage` parameter will not work with the `vsadmin` and `fsxadmin` user accounts. The configuration operation will fail if you specify this parameter.
 
+WARNING: FSx for NetApp ONTAP volumes come preconfigured with a SnapMirror. Deleting PVCs will not succeed, as Trident cannot modify or break them. Users will need to delete their PVs manually and clean them up on FSx.
```

Comment:
@balaramesh technically the official product name is Amazon FSx for ONTAP. You probably don't need "NetApp" before ONTAP.